Update documentation for the charm (#87)
canonical/digest-squid-auth-helper: docs/how-to/manage-users.md

# How to manage users

The following actions can be performed with the users:

## Give access to users

New users can be given access through the `juju run digest-squid-auth-helper/0 create-user username=yourusername`. The command will return you the username, realm and password.

## List current users

> The `juju run digest-squid-auth-helper/0 list-users` will return the list of active users.

Reviewer: Either
```suggestion
Run `juju run digest-squid-auth-helper/0 list-users` to return the list of active users.
```
or
```suggestion
The command `juju run digest-squid-auth-helper/0 list-users` will return the list of active users.
```
(I have a slight preference for the first suggestion, but it's up to you)

## Remove users

> You can remove users with `juju run digest-squid-auth-helper/0 remove-user username=yourusername`

Reviewer: Nitpick, but I think we should be as direct as possible in how-to guides. "You can" implies that there's a choice (maybe there is), and I don't want our readers to have any doubt or uncertainty. 
```suggestion
Remove users with `juju run digest-squid-auth-helper/0 remove-user username=yourusername`
```

Note: Squid caches some authentication lookups (see [documentation](https://wiki.squid-cache.org/Features/Authentication#does-squid-cache-authentication-lookups)).

## Configure the authentication type

The default configuration can be seen using the `juju config digest-squid-auth-helper` command.

> The default authentication type is "digest". If you want to switch to Basic authentication, you can issue the `juju config digest-squid-auth-helper authentication-type=basic` command.

Reviewer: Nitpick
```suggestion
The default authentication type is "digest". If you want to switch to Basic authentication, use the `juju config digest-squid-auth-helper authentication-type=basic` command.
```
